src/main/java/org/reaktivity/nukleus/http2/internal/routable/stream/TargetOutputEstablishedStreamFactory.java

```diff
@@ -223,7 +225,7 @@ private void processBegin(
                 }
 
                 this.streamState = this::afterBeginOrData;
-                this.window = 8192;
+                this.window = WINDOW_SIZE.intValue();
```

Comment:
Even though this is an internal system property, it is the same semantic level as `Configuration` and should be resolved (& parsed) in `NukleusFactorySpi` rather than on each new `Stream`.